pyproject.toml

```diff
@@ -25,7 +25,8 @@ dependencies = [
     "xmltodict",
     "pint",
     "fairmat-readers-xrd",
-    "pynxtools>=0.7.1",
```

Comment:
Remove this once https://github.com/FAIRmat-NFDI/pynxtools/pull/527 is accepted.